pyuvdata/calfits.py

```diff
@@ -413,7 +440,7 @@ def read_calfits(self, filename, run_check=True, check_extra=True,
             self.Nspws = hdr.pop('NAXIS5')
             # add this for backwards compatibility when the spw CRVAL wasn't recorded
             try:
-                spw_array = uvutils.fits_gethduaxis(F[0], 5, strict_fits=strict_fits) - 1
+                spw_array = uvutils.fits_gethduaxis(F[0], 5, strict_fits=strict_fits)
```

Comment:
Good catch!

Reply:
Yeah, I'm not sure how that made it through before, but I had a bug that I was hunting and I found that one (it didn't fix my bug, but I found it somewhere else).